apps/microtvm/zephyr/template_project/microtvm_api_server.py

```diff
@@ -310,18 +314,11 @@ def _get_nrf_device_args(options):
         help=("If given, port number to use when running the local gdbserver."),
     ),
     server.ProjectOption(
-        "nrfjprog_snr",
-        optional=["open_transport"],
-        type="int",
-        default=None,
-        help=("When used with nRF targets, serial # of the attached board to use, from nrfjprog."),
-    ),
-    server.ProjectOption(
-        "openocd_serial",
+        "serial_number",
```

Comment:
`serial_number` is also used by the `flash` method and must be advertised accordingly, so please add the method `flash` to the `optional` list below.